### DISCUSSION TITLE
Multiple watcher emails

### DISCUSSION THREAD

**Discussion by abdallah-odeh** [talsec/Free-RASP-Community#14] [Ideas]
First, Thank you for this plugin!
Is there anyway to add multiple receivers for the weekly reports?

**Comment by msikyna** [comment]
Hello @abdallah-odeh ,
currently there is no way adding multiple receivers for the weekly reports. Adding more email addresses to the **watcherMail** parameter will result in no delivery of the report at all, please do not do that.

However, thank you for an interesting idea! We might consider implementing it in future versions. I will transfer this to the [Ideas](https://github.com/talsec/Free-RASP-Community/discussions/categories/ideas), where it can get upvoted.

Kind regards,
Matúš, Talsec dev

**Reply by abdallah-odeh** [reply]
Thank you @msikyna for the quick response, will be happy if this was implemented soon!
